manager/src/main/java/org/openremote/manager/event/ClientEventService.java

```diff
@@ -209,22 +241,24 @@ public void configure() throws Exception {
                             }
                         })
                         .choice()
-                            .when(exchange -> isGatewayClientId(getClientId(exchange)))
-                                .to(GatewayService.GATEWAY_EVENT_TOPIC)
-                            .otherwise()
-                                .to(ClientEventService.CLIENT_EVENT_TOPIC)
+                            .when(header(Constants.AUTH_CONTEXT).isNotNull())
```

Comment:
I think you should look for the `HEADER_CONNECTION_TYPE` because long term we might have anonymous access for public assets